infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/limits/ram/RamLimitRequestProvisioner.java

```diff
@@ -43,6 +45,10 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
         if (memoryLimitAttribute != null) {
           Containers.addRamLimit(container, Long.parseLong(memoryLimitAttribute));
         }
+        String memoryRequestAttribute = attributes.get(MEMORY_REQUEST_ATTRIBUTE);
+        if (memoryRequestAttribute != null) {
+          Containers.addRamRequest(container, Long.parseLong(memoryRequestAttribute));
```

Comment:
Maybe it makes sense to set Ram requests  to Ram limit if the request is not specified explicitly. WDYT?

Reply:
It seems that we need to rewrite this class a bit. Or add another one that would ensure that all the machines have memory request and limit in place. I created an issue for that https://github.com/eclipse/che/issues/11428